react/components/EXPERIMENTAL_Table/README.md

```diff
@@ -23,123 +24,300 @@ Example with simple structure:
+    id: 'email',
+    title: 'Email',
+    /** Custom renderer for email prop */
+    headerRenderer: ({ columnData: { title } }) => {
+      return (
+        <span>
+          <Emoji symbol="💌" label="mail" /> {title}
+        </span>
+      )
+    },
+  },
+  {
+    /** Definitions for the age prop */
+    id: 'age',
+    title: 'Age',
+    /** Custom renderer for age prop */
+    cellRenderer: ({ cellData: age }) => {
+      const bgColor = age > 18 ? '#F71963' : '#134CD8'
+      return (
+        <Tag color="#FFFFFF" bgColor={bgColor}>
+          {age} years
+        </Tag>
+      )
+    },
+  },
+  {
+    /** Definitions for the country prop */
+    id: 'country',
+    /** This means that the title shown for the country property will be Nationality */
+    title: 'Nationality',
+  },
+]
+```
 
-### Example Of Usage
+##### Working example:
 
 ```js
-// Imports
 const useTableMeasures = require('./hooks/useTableMeasures.tsx').default
 const Tag = require('../Tag/index.js').default
 
-// Define the columns
-const columns = [
+const heroColumns = [
   {
     id: 'name',
     title: 'Name',
   },
   {
     id: 'email',
     title: 'Email',
+    headerRenderer: ({ columnData: { title } }) => {
+      return (
+        <span>
+          <Emoji symbol="💌" label="mail" /> {title}
+        </span>
+      )
+    },
   },
   {
-    id: 'number',
-    title: 'Number',
-    cellRender: ({ cellData }) => {
-      return <Tag>{cellData}</Tag>
+    id: 'age',
+    title: 'Age',
+    cellRenderer: ({ cellData: age }) => {
+      const bgColor = age > 18 ? '#F71963' : '#134CD8'
+      return (
+        <Tag color="#FFFFFF" bgColor={bgColor}>
+          {age} years
+        </Tag>
+      )
     },
   },
   {
     id: 'country',
-    title: 'Country',
+    title: 'Nationality',
   },
 ]
 
-// Define the items
-const items = [
+const heroes = [
   {
     name: "T'Chala",
     email: 'black.panther@gmail.com',
-    number: 1.88191,
+    age: 31,
     country: '🇰🇪Wakanda',
   },
   {
     name: 'Peter Parker',
     email: 'spider.man@gmail.com',
-    number: 3.09191,
+    age: 17,
     country: '🇺🇸USA',
   },
   {
-    name: 'Shang-Chi',
-    email: 'kungfu.master@gmail.com',
-    number: 39.09222,
-    country: '🇨🇳China',
+    name: 'Natasha Romanoff',
+    email: 'black.widow@gmail.com',
+    age: 29,
+    country: '🇷🇺Russia',
+  },
+]
+
+function Emoji({ symbol, label = '' }) {
+  return (
+    <span role="img" arial-label={label} aria-hidden={label ? 'false' : 'true'}>
+      {symbol}
+    </span>
+  )
+}
+
+function SimpleExample() {
+  const measures = useTableMeasures({
+    size: heroes.length,
+  })
+
+  return <Table measures={measures} columns={heroColumns} items={heroes} />
+}
+;<SimpleExample />
+```
+
+##### Toggle visibility
+
+It is possible to hide/show columns. This can be done using the `EXPERIMENTAL_useTableVisibility` hook and the `Columns` button (part of the `Toolbar`). Check the working example below:
```

Comment:
```suggestion
It is possible to show/hide columns. This can be done using the `EXPERIMENTAL_useTableVisibility` hook and the `Columns` button (part of the `Toolbar`). Check the working example below:
```